ts4.1/icu.d.ts

```diff
@@ -0,0 +1,85 @@
+declare module 'react-i18next/icu.macro' {
+  type Namespace = import('.').Namespace;
+  type DefaultNamespace = import('.').DefaultNamespace;
+  type Resources = import('.').Resources;
+  type TFuncKey<N extends Namespace = DefaultNamespace, T = Resources> = import('.').TFuncKey<N, T>;
+  type i18n = import('i18next').i18n;
+
+  export interface PluralSubProps<
+    K extends TFuncKey<N> extends infer A ? A : never,
+    N extends Namespace = DefaultNamespace
+  > {
+    children?: never;
```

Comment:
Here we can simply remove children

Reply:
the intent here was to prevent children. In other words, typescript will error if you pass children in, as a way of signaling "children will be ignored"

Reply:
I believe this will happen for `Select` component only 🤔 , because you're intersecting its props with `SelectSubProps`

Reply:
true, I'll add it to PluralProps to solve that oversight